### ISSUE TITLE
Extra quotes added when rebuilding rule

### ISSUE THREAD

**Issue by hegga** [plyara/plyara#54]
Hi,

When parsing a rule with a specified, but empty description field, like this:

```
rule sample
{
    meta:
        description = ""
        author = "me"
    strings:
        $ = { 01 02 03 04 } // string 1
        $ = { 01 02 03 05 } // string 2
    condition:
        all of ($) // condition
}
```

plyara.Plyara.rebuild_yara_rule returns too many quotes, like this:

```
rule sample
{
    meta:
       description = """"
        author="me"
    strings:
        $ = { 01 02 03 04 } // string 1
        $ = { 01 02 03 05 } // string 2
    condition:
        all of ($) // condition
}
```

I'm running version 2.0.1

**Comment by rshipp** [comment]
Thanks for the issue! Looks like a quick fix (#55) - I'll get a new version pushed out as soon as someone has had a chance to review it.

**Comment by rshipp** [comment]
Alright, pushed new version 2.0.2 to PyPI to address this. Feel free to let us know if you find anything else!
